adabot_description/launch/adabot.model.launch

```diff
@@ -35,6 +37,7 @@
     <arg name="scale"        default="4" />
 
     <!-- Store arguments on the parameter server for nodes -->
+    <param name="debug_visual" value="$(arg debug_visual)" />
```

Comment:
I can't think of a reason that we'd need to put this parameter on the parameter server. Remove this or convince me otherwise.